extensions/resteasy-reactive/quarkus-resteasy-reactive/spi-deployment/src/main/java/io/quarkus/resteasy/reactive/server/spi/CopyVertxContextDataBuildItem.java

```diff
@@ -0,0 +1,10 @@
+package io.quarkus.resteasy.reactive.server.spi;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+/**
+ * A marker build item which indicates that the current Vertx request context data needs
+ * to be copied into the connection context
+ */
+public final class CopyVertxContextDataBuildItem extends MultiBuildItem {
```

Comment:
Wondering if this should be part of the vert.x extension instead, even if it is only consumed in resteasy reactive now.

Reply:
We can always move it to Vert.x (which has a much wider scope) if necessary.

I think keeping it contained as much as possible make most sense for now